sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -84,6 +84,9 @@ public interface ContainerGroup extends GroupableResource<ContainerInstanceManag
     /** @return the DNS configuration for the container group */
     DnsConfiguration dnsConfig();
 
+    /** @return the reuse policy of DSN name label */
+    DnsNameLabelReusePolicy dnsNameLabelReusePolicy();
```

Comment:
should conform to the setter method's name

Reply:
Fixed in the new version.